scripts/simple-tcp-server-client.py

```diff
@@ -0,0 +1,307 @@
+#!/bin/python3
```

Comment:
Are we sure there is not like a iperf like python library we can use? Or do you think writing the code from a low level has some advantages?

Reply:
I think the main point of this PR is to add all the basics for a python-based tool.

It works as is, but probably does not something very useful.

The use is to have some code in place, that can be modified and adjusted when we have a particular use.

Maybe an iperf-like library would be useful, and then the script should be easily extendable to support that, once we have the exact use-case.

Reply:
Ah ok thanks!